src/containers/BooksList/BooksList.js

```diff
@@ -0,0 +1,19 @@
+/* eslint-disable no-console */
+import { useState } from 'react';
```

Comment:
- [ ] <b>REQUIRED:</b> This `eslint-disable` comment is useless in this context. kindly remove it.